src/ActivityRepository.js

```diff
@@ -103,25 +103,16 @@ class ActivityRepository {
     return this.user.slice(index - 6, index + 1);
   }
 
-  getPositiveStepTrends() {
+  getStepTrends(positive) {
     return this.user.reduce((acc, day, index) => {
       if (index < 2) {
         return acc;
       }
-      if ((day.numSteps > this.user[index - 1].numSteps) &&
+      if (positive && (day.numSteps > this.user[index - 1].numSteps) &&
         (this.user[index - 1].numSteps > this.user[index - 2].numSteps)) {
         acc.push(day.date);
-      }
-      return acc;
-    }, []);
-  }
-
-  getNegativeStepTrends() {
-    return this.user.reduce((acc, day, index) => {
-      if (index < 2) {
-        return acc;
-      }
-      if ((day.numSteps < this.user[index - 1].numSteps) &&
+      } 
+      if (!positive && (day.numSteps < this.user[index - 1].numSteps) &&
         (this.user[index - 1].numSteps < this.user[index - 2].numSteps)) {
         acc.push(day.date);
       }
```

Comment:
I'm a little confused to see my activity repo changes here - was your master not up to date?